stl/inc/regex

```diff
@@ -2428,6 +2485,8 @@ public:
                && _MyVal._At(0) == _Right._MyVal._At(0);
     }
 
+    // TRANSITION, VSO-900973
+    // Overload resolution prefers implicit conversions over re-writes

```

Comment:
Ditto "How does the bug manifest here given that `_Iterator_base` has no comparisons?"